spec/policies/document_policy_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe DocumentPolicy do
+  let(:cma_organisation_id) { CmaCase.organisations.first }
+  let(:gds_editor) { User.new(permissions: %w(signin gds_editor)) }
+  let(:cma_departmental_editor) { User.new(permissions: %w(signin editor), organisation_content_id: cma_organisation_id) }
+  let(:cma_writer) { User.new(permissions: ['signin'], organisation_content_id: cma_organisation_id) }
+
+  permissions :index?, :show?, :new?, :create?, :edit?, :update? do
+    it 'denies access to users from another organisation' do
+      expect(described_class).not_to permit(cma_writer, AaibReport)
+    end
+
+    it 'grants access to users from the organisation to which the document belongs' do
+      expect(described_class).to permit(cma_writer, CmaCase)
+    end
```

Comment:
Without wishing to bikeshed could we make these more explicit by using spec-local Document subclasses that explicitly have different sets of organisations in them?  To understand this spec I have to go and look at the implementations of `AaibReport` and `CmaCase` to see that they have different organisations - if they were spec-local it would all be right here.